snippets/standard/data/sqlite/AggregateFunctionSample/Program.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using Microsoft.Data.Sqlite;
+
+namespace AggregateFunctionSample
+{
+    class Program
+    {
+        static void Main()
+        {
+            var connection = new SqliteConnection("Data Source=:memory:");
+            connection.Open();
+
+            var createCommand = connection.CreateCommand();
+            createCommand.CommandText =
+            @"
+                CREATE TABLE student (
+                    gpa REAL
+                );
+
+                INSERT INTO student
+                VALUES (4.0),
+                       (3.0),
+                       (3.0),
+                       (2.0),
+                       (2.0),
+                       (2.0),
+                       (2.0),
+                       (1.0),
+                       (1.0),
+                       (0.0);
+            ";
+            createCommand.ExecuteNonQuery();
+
+            #region snippet_CreateAggregate
+            connection.CreateAggregate(
+                "stdev",
+
+                // A tuple to maintain context between rows
+                (Count: 0, Mean: 0.0, Sum: 0.0),
+
+                // This is called for each row
+                ((int Count, double Mean, double Sum) context, double value) =>
+                {
+                    context.Count++;
+                    var delta = value - context.Mean;
```

Comment:
I think this is mathematically wrong. The `context.Mean` gets updated each row, so the `delta` would be accurate only for the final row.  

See [this example](https://docs.microsoft.com/en-us/dotnet/csharp/tuples#tuples-as-method-return-values) for an alternative algorithm that uses one enumeration for StdDev.

Reply:
Copied the example's algorithm. The old result is definitely incorrect. Thanks! lol, that's what I get for grabbing some arbitrary algorithm off Wikipedia... 😉